packages/cli/src/commands/multi-chain/deploy.ts

```diff
@@ -71,7 +72,8 @@ export default class MultiChainDeploy extends Command {
     flags.projectName = await valueOrPrompt(flags.projectName, 'Enter project name', 'Project name is required');
 
     // Multichain query descriptor
-    const ipfsCID = fileToCidMap.get(path.basename(multichainManifestPath));
+    const ipfsCID = Array.from(fileToCidMap).find(([file]) => file === '')?.[1];
```

Comment:
Is this finding the directory? Its hard to tell from the code, maybe we could have a comment to explain. Could there also be some validation on the key from the map?

Reply:
Yes, finding the directory